Apres_Ski_API/fixtures/restaurant.json

```diff
@@ -0,0 +1,325 @@
+[
+  {
+    "model": "Apres_Ski_API.restaurant",
+    "pk": 1,
+    "fields": {
+      "name": "9600 Kitchen",
+      "address": "550 Village Rd Breckenridge, CO 80424",
+      "food_type": "American",
+      "cost": "2",
+      "vibe": "['Casual', 'Upscale']",
```

Comment:
This was a compromise when we moved the `engagements` table to an extension (that table also records vibe). That way FE could still have a collection of data on the vibe of a restaurant. 

We can instead just record a single vibe here.

The engagements table is already made and associated with restaurants and users.

Reply:
Talking with FE about this. We will replace the data stored in the `vibe` column to the `engagement` table.

This will be stored as a single string. The idea being that a user can rate a restaurant from one of a preset of list of vibes. Then their selection would be recorded.